check_and_gen.py

```diff
@@ -10,9 +10,9 @@
 auth = ""
 keys = ""
 already_exists = False
-if isfile('data.txt'):
+if isfile(cfg['data_path']):
```

Comment:
Why was this wrapped in a `cfg[]`? What does that mean?

Reply:
The path was hardcoded in several places (not only on this script) and so I just moved it to the `cfg` object so that it can be referenced in those places.